docker/scvi/scripts/main/process.py

```diff
@@ -35,7 +35,7 @@
     "--marker-genes",
     dest="marker_genes",
     type=str,
-    default="resources/celltype_marker_table.csv",
+    default="resources/celltype_marker_table.csv", # this doesn't seem accurate
```

Comment:
@kfang4 I haven't tracked down the _source-of-truth_ for this reference.  But I remember we moved these out of resources since they ought to be parameters.   Should we just remove the default?

Reply:
The best fallback reference would be  https://github.com/NIH-CARD/brain-taxonomy/blob/main/markers/cellassign_card_markers.csv

Reply:
I think this is your local path when you first wrote the script. I can remove the defaults.

Reply:
Removed default.